patches/services-network-tcp_connected_socket.cc.patch

```diff
@@ -1,22 +0,0 @@
-diff --git a/services/network/tcp_connected_socket.cc b/services/network/tcp_connected_socket.cc
-index d8d6a1cd98c81268ab4fb597f682d71664317134..335859f434e3b7822ab483577a25a7da5586b8ec 100644
---- a/services/network/tcp_connected_socket.cc
```

Comment:
Chromium upstream already has this code exactly now.

Reply:
yep, that's expected. cc @yrliou